### DISCUSSION TITLE
tickInterval가 readonly인데 따로 설정할 수 있나요?

### DISCUSSION THREAD

**Discussion by hanaaple** [naverz/zepeto-studio-kor#780] [5. Q&A]
TickInterval가 readonly입니다.
OnTick을 원하는 시간마다 사용하고 싶은데 가능할까요?

**Comment by minjujuu** [comment]
서버 코드에서 다음과 같이 사용하시면 원하는 시간마다 실행되도록 하실 수 있을 것 같아요!
```
 onTick(deltaTime: number) {
        this.tempTime += deltaTime;

        /** 예시: spawnDelayTime 마다 몬스터가 생성되도록 함 */
        if (this.tempTime / 1000 > this.spawnDelayTime) {
                 this.tempTime = 0;
        }
```

**Comment by smilejsu82** [comment]
> 서버 코드에서 다음과 같이 사용하시면 원하는 시간마다 실행되도록 하실 수 있을 것 같아요!
> 
> ```
> 
>  onTick(deltaTime: number) {
> 
>         this.tempTime += deltaTime;
> 
> 
> 
>         /** 예시: spawnDelayTime 마다 몬스터가 생성되도록 함 */
> 
>         if (this.tempTime / 1000 > this.spawnDelayTime) {
> 
>                  this.tempTime = 0;
> 
>         }
> 
> ```

저도 서버에서 setInterval 안되서 고민중이였는데 감사합니다 ^^
